File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.py
def solution(rank, attendance):
    answer = 0
    candidates = []
    
    for i in range(len(attendance)):
        if attendance[i]:
            candidates.append((rank[i], i))
                            
    candidates.sort()
    
    a = candidates[0][1]
    b = candidates[1][1]
    c = candidates[2][1]
    
    answer = 10000 * a + 100 * b + c
    return answer

File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.js
function solution(rank, attendance) {
    let topRank = []
    
    for(let i=0; i<attendance.length; i++){
        if(attendance[i]) topRank.push([rank[i], i]);
    }
    
    topRank.sort((a,b) => a[0] - b[0]);

    let a = topRank[0][1]
    let b = topRank[1][1]
    let c = topRank[2][1]

    return 10000*a+100*b+c
}